pkg/cvo/metrics.go

```diff
@@ -297,6 +310,23 @@ func (m *operatorMetrics) Collect(ch chan<- prometheus.Metric) {
 		g.Set(1.0)
 		ch <- g
 	}
+
+	// check ability to retrieve recommended updates
+	if availableUpdates := m.optr.getAvailableUpdates(); availableUpdates != nil {
+		g2 := m.clusterVersionOperatorUpdateRetrievalTimestampSeconds.WithLabelValues("")
```

Comment:
can we shift this `g2` bit down to just before we set it from `LastSyncOrConfigChange`?  No need to split the `g2` handling in half and drop the failure-reason stuff in the middle.